2021/34xxx/CVE-2021-34604.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
An exception of type JSONDecodeError occurred. Arguments:

Reply:
fixed in 2nd commit